ethcore/src/state.rs

```diff
@@ -339,6 +339,18 @@ impl fmt::Debug for State {
 	}
 }
 
+impl Clone for State {
```

Comment:
Where is this used? I implemented state snapshot specifically to avoid state cloning, which is quite expensive.